Objects/Objects/BuiltElements/Revit/RevitToposolid.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using Objects.Geometry;
+using Objects.Utils;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+
+namespace Objects.BuiltElements.Revit
+{
+  public class RevitToposolid : Base, IDisplayValue<List<Mesh>>
+  {
+    public RevitToposolid() { }
+
+    [SchemaInfo("RevitToposolid", "Creates a Revit Toposolid", "BIM", "Architecture")]
```

Comment:
Sorry for being a PITA 🙇🏼‍♂️ 3rd parameter in the SchemaInfo should be "Revit", or this will land in the wrong tab in Grasshopper.